bitcoin/examples/sign-tx-segwit-v0.rs

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: CC0-1.0
+
+//! Sign a transaction that spends an p2wpkh unspent output.
```

Comment:
nit use a not an here. Sign a transaction that spends a p2wpkh unspent output.  Not that I'm a grammar expert at all, but I think the only time to use an is when the following word begins with a vowel, like "an output".

Reply:
You are correct, cheers. Will fix.

Reply:
This change is from #2095, I've fixed it there.